packages/merge-on-green/src/merge-logic.ts

```diff
@@ -106,12 +104,9 @@ mergeOnGreen.getPR = async function getPR(
       repo,
       pull_number: pr,
     });
-    console.log(
-      `github's determination of whether the ${owner}/${repo}/${pr} is mergeable: ${data.data.mergeable} ${data.data.mergeable_state}`
-    );
     return data.data;
   } catch (err) {
-    return { title: '', body: '' };
+    return { title: '', body: '', mergeable: false, mergeable_state: '' };
```

Comment:
I like this step, where if you couldn't get the data you indicate it's not `mergeable`.